pkg/bootstrap/docker/up.go

```diff
@@ -511,6 +518,38 @@ func (c *ClientStartConfig) EnsureHostDirectories(io.Writer) error {
 	return c.HostHelper().EnsureVolumeShare()
 }
 
+// EnsureDefaultRedirectURIs merges a default URL to an auth client's RedirectURIs array
+func (c *ClientStartConfig) EnsureDefaultRedirectURIs(out io.Writer) error {
+	oc, _, err := c.Clients()
+	if err != nil {
+		return nil
+	}
+
+	webConsoleOAuth, err := oc.OAuthClients().Get(defaultRedirectClient)
```

Comment:
if the web console OAuth client doesn't exist, I would exit early, rather than error